sympy/matrices/matrices.py

```diff
@@ -4278,59 +4278,239 @@ def entry_U(i, j):
+        is the notation that SymPy is using.
+
+        PLU decomposition is a decomposition of a $m, n$ matrix $A$ in
+        the form of $P A = L U$ where
+
+        * $L$ is a $m, m$ lower triangular matrix with unit diagonal
+          entries.
+        * $U$ is a $m, n$ upper triangular matrix.
+        * $P$ is a $m, m$ permutation matrix.
+
+        So, for a square matrix, the decomposition would look like:
+
+        .. math::
+            L = \begin{bmatrix}
+            1 & 0 & 0 & \cdots & 0 \\
+            L_{1, 0} & 1 & 0 & \cdots & 0 \\
+            L_{2, 0} & L_{2, 1} & 1 & \cdots & 0 \\
+            \vdots & \vdots & \vdots & \ddots & \vdots \\
+            L_{n-1, 0} & L_{n-1, 1} & L_{n-1, 2} & \cdots & 1
+            \end{bmatrix}
+
+        .. math::
+            U = \begin{bmatrix}
+            U_{0, 0} & U_{0, 1} & U_{0, 2} & \cdots & U_{0, n-1} \\
+            0 & U_{1, 1} & U_{1, 2} & \cdots & U_{1, n-1} \\
+            0 & 0 & U_{2, 2} & \cdots & U_{2, n-1} \\
+            \vdots & \vdots & \vdots & \ddots & \vdots \\
+            0 & 0 & 0 & \cdots & U_{n-1, n-1}
+            \end{bmatrix}
+
+        And for a matrix with more rows than the columns,
+        the decomposition would look like:
+
+        .. math::
+            L = \begin{bmatrix}
+            1 & 0 & 0 & \cdots & 0 & 0 & \cdots & 0 \\
+            L_{1, 0} & 1 & 0 & \cdots & 0 & 0 & \cdots & 0 \\
+            L_{2, 0} & L_{2, 1} & 1 & \cdots & 0 & 0 & \cdots & 0 \\
+            \vdots & \vdots & \vdots & \ddots & \vdots & \vdots & \ddots
+            & \vdots \\
+            L_{n-1, 0} & L_{n-1, 1} & L_{n-1, 2} & \cdots & 1 & 0
+            & \cdots & 0 \\
+            L_{n, 0} & L_{n, 1} & L_{n, 2} & \cdots & L_{n, n-1} & 1
+            & \cdots & 0 \\
+            \vdots & \vdots & \vdots & \ddots & \vdots & \vdots
+            & \ddots & \vdots \\
+            L_{m-1, 0} & L_{m-1, 1} & L_{m-1, 2} & \cdots & L_{m-1, n-1}
+            & 0 & \cdots & 1 \\
+            \end{bmatrix}
+
+        .. math::
+            U = \begin{bmatrix}
+            U_{0, 0} & U_{0, 1} & U_{0, 2} & \cdots & U_{0, n-1} \\
+            0 & U_{1, 1} & U_{1, 2} & \cdots & U_{1, n-1} \\
+            0 & 0 & U_{2, 2} & \cdots & U_{2, n-1} \\
+            \vdots & \vdots & \vdots & \ddots & \vdots \\
+            0 & 0 & 0 & \cdots & U_{n-1, n-1} \\
+            0 & 0 & 0 & \cdots & 0 \\
+            \vdots & \vdots & \vdots & \ddots & \vdots \\
+            0 & 0 & 0 & \cdots & 0
+            \end{bmatrix}
+
+        Finally, for a matrix with more columns than the rows, the
+        decomposition would look like:
+
+        .. math::
+            L = \begin{bmatrix}
+            1 & 0 & 0 & \cdots & 0 \\
+            L_{1, 0} & 1 & 0 & \cdots & 0 \\
+            L_{2, 0} & L_{2, 1} & 1 & \cdots & 0 \\
+            \vdots & \vdots & \vdots & \ddots & \vdots \\
+            L_{m-1, 0} & L_{m-1, 1} & L_{m-1, 2} & \cdots & 1
+            \end{bmatrix}
```

Comment:
Was there ever any conclusion about using latex in docstrings? This looks good in the html but is unreadable here.

Reply:
https://docs.sympy.org/1.5.1/documentation-style-guide.html#code
It says like
> If the sentence in question is discussing mathematics, then LaTeX should be used, but if the sentence is discussing the SymPy implementation specifically, then code should be used.